sdk/documentintelligence/azure-ai-documentintelligence/azure/ai/documentintelligence/_model_base.py

```diff
@@ -462,7 +470,13 @@ def _get_rest_field(
 
 
 def _create_value(rf: typing.Optional["_RestField"], value: typing.Any) -> typing.Any:
-    return _deserialize(rf._type, value) if (rf and rf._is_model) else _serialize(value, rf._format if rf else None)
+    if not rf:
+        return _serialize(value, None)
+    if rf._is_multipart_file_input:
```

Comment:
Don't see this added in Java generation, was this anything specific to python?

Reply:
It's python specific.